docs/flashbots-mev-share/searchers/tutorials/flash-loan-arbitrage/bot.mdx

```diff
@@ -0,0 +1,126 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Automated Arbitrage Bot
+
+With the flash-loan arbitrage smart contract finished, our last task is to automate the process of finding and back-running other users' trades.
+
+To reiterate our goal, the idea is to find pending transactions from other Ethereum users and backrun them (send our transaction immediately after the user's transaction). If the user's trade moves the price enough, we'll arbitrage the trading pair between two exchanges for a profit.
+
+## Finding Pending Transactions
+
+When a new transaction is received, it's broadcast to searchers via [Event Stream](/flashbots-mev-share/searchers/event-stream). Each transaction shares varying amounts of data via [hints](#/), but by default, transactions that trade on Uniswap, Balancer, or Curve will expose the trading pair's contract address in the log topics.
```

Comment:
Specify "when a new transaction is received by MEV-Share" or insert "MEV-Share" somewhere here

Reply:
good look, reworded